src/ecdsa/test_pyecdsa.py

```diff
@@ -1339,3 +1339,68 @@ def test_brainpoolP512r1(self):
             y_Z=int("7DB71C3DEF63212841C463E881BDCF055523BD368240E6C3143BD8DEF8"
                     "B3B3223B95E0F53082FF5E412F4222537A43DF1C6D25729DDB51620A83"
                     "2BE6A26680A2", 16))
+
```

Comment:
extra line

Reply:
https://www.python.org/dev/peps/pep-0008/: "Surround top-level function and class definitions with two blank lines."